source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -93,22 +127,30 @@ Strategies
        | Deletes at most one document that matches your sink connector's
          key structure by the ``_id`` field only when the document
          contains a null value structure.
-       | This is implicitly specified when you set ``mongodb.delete.on.null.values=true``.
-       | You can set this explicitly with the following configuration: 
+       |
+       | This is the default value for the
+         ``delete.writemodel.strategy`` configuration property.
+       |
+       | This strategy is implicitly selected as the value of the
```

Comment:
Suggestion:

I think "default value" could be easier than "implicitly selected...value" to understand.

```suggestion
       | This strategy is set as the default value of the
```